pkg/controllers/configuration.go

```diff
@@ -218,18 +218,23 @@ func (r *HypershiftDeploymentReconciler) ensureConfiguration(ctx context.Context
 			k := genKey(se.secretRef, hyd)
 			secret, err := r.generateSecret(ctx, k, overrideNamespace(helper.GetHostingNamespace(hyd)))
 			if err != nil {
-				r.Log.Info(fmt.Sprintf("did not find and copy secret %s: %s", k, err.Error()))
+				r.Log.V(4).Info(fmt.Sprintf("did not find and copy secret %s: %s", k, err.Error()))
```

Comment:
What level is 4? We haven't used this anywhere else in the code, how would one leverage it?

Reply:
Not sure what is the common way.

One way is to update the csv or deployment and add `- --v=4` to the command. 

Saw in the hypershift code they have a lot of debugs in v(4), so thought maybe we could have more verbose logs for v(4) in case we need to troubleshoot. 

Would v(1) be better? Definitely don't want to in just Info though.

Reply:
So you can't do much with our deployments because its reconciled by the mce operator. So if there is no way to change the setting....